aiogram/dispatcher/filters/builtin.py

```diff
@@ -728,18 +728,20 @@ def __init__(self, chat_type: typing.Container[ChatType]):
 
         self.chat_type: typing.Set[str] = set(chat_type)
 
-    async def check(self, obj: Union[Message, CallbackQuery, ChatMemberUpdated]):
+    async def check(self, obj: Union[Message, CallbackQuery, ChatMemberUpdated, InlineQuery]):
         if isinstance(obj, Message):
-            obj = obj.chat
+            chat_type = obj.chat.type
         elif isinstance(obj, CallbackQuery):
-            obj = obj.message.chat
+            chat_type = obj.message.chat.type
         elif isinstance(obj, ChatMemberUpdated):
-            obj = obj.chat
+            chat_type = obj.chat.type
+        elif isinstance(obj, InlineQuery):
```

Comment:
```python
... and obj.chat_type
```
because `chat_type` is optional field and can be `None`, we don't need additional actions to check the `self.chat_type` contains `None` or not.

Reply:
I did a chat type check in a secret chat and it is really not necessary to do a check, since the result of the type will be `None` and there will be no error, if i understood you properly.